cmd/performance-profile-creator/cmd/root.go

```diff
@@ -578,7 +578,7 @@ type ProfileCreatorArgs struct {
 	EnableHardwareTuning        bool   `json:"enable-hardware-tuning,omitempty"`
 }
 
-func createProfile(profileData ProfileData) error {
+func makePerformanceProfileFrom(profileData *ProfileData) *performancev2.PerformanceProfile {
```

Comment:
if we don't mutate `profileData`, and it seems to me we don't it seems better to pass a copy rather than a pointer

Reply:
Done. thanks